llvm/test/CodeGen/SPIRV/hlsl-intrinsics/firstbithigh.ll

```diff
@@ -0,0 +1,107 @@
+; RUN: llc -verify-machineinstrs -O0 -mtriple=spirv-unknown-unknown %s -o - | FileCheck %s
+; RUN: %if spirv-tools %{ llc -O0 -mtriple=spirv-unknown-unknown %s -o - -filetype=obj | spirv-val %}
+
+; CHECK: OpMemoryModel Logical GLSL450
+
+define noundef i32 @firstbituhigh_i32(i32 noundef %a) {
```

Comment:
This file has a bug
project/llvm/test/CodeGen/SPIRV/hlsl-intrinsics/firstbithigh.ll -o - -filetype=obj
error: line 54: Illegal number of components (1) for TypeVector
  %v1uint = OpTypeVector %uint 1

Reply:
Yes I saw I am working on fixing it.